### ISSUE TITLE
Chorioamniotic separation 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7079]
**Preferred term label:**
Chorioamniotic separation 

**Synonyms**
Separation of the chorionic and amniotic membranes

**Definition (free text, please give PubMed ID)**

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5039039/

**Parent term (use hpo.jax.org/app)**
Abnormal placental membrane morphology HP:0011409  ?? is new hierarchy needed

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ichthyosis prematurity syndrome | 608649

https://pubmed.ncbi.nlm.nih.gov/21856041/

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@sparkste, @rhiannonmellis -Good day! We are all slowly trickling back from well needed vacation breaks, hoping to dive back into our prenatal project! If you could review, and comment on the above term, it would be greatly appreciated. Thank you in advance for your cooperation and help.

**Comment by sparkste** [comment]
Great term to add.

I do think it is important to differentiate chorioamniotic separation that results from a procedure (ie amniocentesis) though, versus spontaneous chorioamniotic separation that is associated with a disease process.

Spontaneous chorioamniotic separation may also result from lethal restrictive dermopathy (OMIM 275210).

Thanks!
Teresa



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by pnrobinson** [comment]
label: Spontaneous chorioamniotic separation
definition: Persistence separation of the chorionic and amnionic membranes after the 16th week of gestation (not as a result of a procedure such as amniocentesis).
Comment: The separation of the chorion and amnion before 14 weeks' of gestation is physiologically normal. The amnion and chorion usually fuse between 14 and 16 weeks, and any chorioamniotic separation (CAS) that persists after 16 weeks is uncommon and anomalous. CAS can occur spontaneously or after an intrauterine intervention such as amniocentesis, fetal blood sampling, or fetal surgery.
 PMID:27683622

**Comment by pnrobinson** [comment]
parent -- Abnormal placental membrane morphology  is good

**Comment by pnrobinson** [comment]
@matentzn we should discuss this at an upcoming Monarch meeting. The participants in the workgroup today noted that some prenatal phenotypes are only abnormal after or before a certain gestational age. It seems to be it would be useful to introduce either way of representing this in the logical definition or to use semantic annotations that downstream software could use for interpretation.

**Comment by pnrobinson** [comment]
adding term
